benchmark/notes.md

```diff
@@ -52,7 +52,7 @@ installed with homebrew.
 
 ```
 git checkout v0.30.0
-cargo build --release
+cargo build --profile bench
```

Comment:
thank you!

however, i did this a bit differently because i wanted to keep the record of what i did for now.
https://github.com/yamt/toywasm/commit/1153d5b1b1c41ffb5292473f8acdf5f94a8ba08f
when i run it next time, i will update the doc in-place as this PR.